src/component/2d/Left1DChart.tsx

```diff
@@ -40,18 +41,15 @@ function Left1DChart({
 
       const lastXIndex = pathPoints.x.length - 1;
       const lastYIndex = pathPoints.y.length - 1;
-      let path = `M  ${scaleY(pathPoints.y[lastYIndex])} ${scaleX(
-        lastXIndex,
-      )} `;
-      // TODO: use d3-path and remove type assertion.
-      path += (pathPoints.x.slice(0, lastXIndex) as number[]).reduceRight(
-        (accumulator, point, index) => {
-          accumulator += ` L  ${scaleY(pathPoints.y[index])} ${scaleX(point)}`;
-          return accumulator;
-        },
-        '',
-      );
-      return path;
+
+      const pathBuilder = new PathBuilder();
+      pathBuilder.moveTo(scaleY(pathPoints.y[lastYIndex]), scaleX(lastXIndex));
```

Comment:
Those 2 variables lastYindex and lastXIndex are rather useless and I would simply but the corresponding `pathPoints.x.length - 1` directly here.